### PR TITLE
SRV_Channel: separate output functions by vehicle type

### PR DESCRIPTION
**Pull request by Hwurzburg**
while @rmackay9 and @magicrub  have vetted the separation via a list of deletions per vehicle that I had asked them to review...it would not hurt for another pair of eyes to review this

### CODE REVIEW

File: libraries/SRV_Channel/SRV_Channel.cpp [hunk 1]
```diff
@@ -66,6 +66,9 @@ const AP_Param::GroupInfo SRV_Channel::var_info[] = {
...
+    // @Values{Copter}: 0:Disabled,1:RCPassThru,6:mount_pan,7:mount_tilt,8:mount_roll,9:mount_open,10:camera_trigger,12:mount2_pan,13:mount2_tilt,14:mount2_roll,15:mount2_open,22:SprayerPump,23:SprayerSpinner,27:Parachute,28:Gripper,29:LandingGear,30:EngineRunEnable,31:HeliRSC,32:HeliTailRSC,33:Motor1,34:Motor2,35:Motor3,36:Motor4,37:Motor5,38:Motor6,39:Motor7,40:Motor8,51:RCIN1,52:RCIN2,53:RCIN3,54:RCIN4,55:RCIN5,56:RCIN6,57:RCIN7,58:RCIN8,59:RCIN9,60:RCIN10,61:RCIN11,62:RCIN12,63:RCIN13,64:RCIN14,65:RCIN15,66:RCIN16,67:Ignition,69:Starter,81:BoostThrottle,82:Motor9,83:Motor10,84:Motor11,85:Motor12,88:Winch,90:CameraISO,91:CameraAperture,92:CameraFocus,93:CameraShutterSpeed,94:Script 1,95:Script 2,96:Script 3,97:Script 4,98:Script 5,99:Script 6,100:Script 7,101:Script 8,102:Script 9,103:Script 10,104:Script 11,105:Script 12,106:Script 13,107:Script 14,108:Script 15,109:Script 16,120:NeoPixel1,121:NeoPixel2,122:NeoPixel3,123:NeoPixel4,124:RateRoll,125:RatePitch,126:RateThrust,127:RateYaw,129:ProfiLED 1,130:ProfiLED 2,131:ProfiLED 3,132:ProfiLED Clock,133:Winch Clutch,134:SERVOn_MIN,135:SERVOn_TRIM,136:SERVOn_MAX
```
Comment (IamPete1) [inline libraries/SRV_Channel/SRV_Channel.cpp:70]: Copter can do `73:ThrottleLeft,74:ThrottleRight,75:TiltMotorFrontLeft,76:TiltMotorFrontRight` as a bicopter.

Comment (Hwurzburg) [reply libraries/SRV_Channel/SRV_Channel.cpp:70]: thanks...fixed

**Comment by Hwurzburg** [comment]
@rmackay9 I have double checked the plane list...when you have done the copter and rover list, approve and merge....thanks

**Comment by rmackay9** [comment]
As a side note, the parameter descriptions are for users so we should try to keep them human readable.  For example "mount_tilt" should really be "Mount Tilt".  I know many of us are software developers but let's try and at least emulate humans.

**Comment by Hwurzburg** [comment]
be happy to update those as part of this

**Comment by rmackay9** [comment]
These can be removed from Rover: 24:FlaperonLeft, 25:FlaperonRight, 124:RateRoll, 125:RatePitch, 126:RateThrust, 127:RateYaw
These can be removed from both Rover and Copter: 67:Ignition, 69:Starter

@Hwurzburg, ok great, thanks.  I'll leave it to you whether you want to correct the names as part of this PR or something else.  If we want to be really nitpicky we could decide whether we want spaces between words or not.  I suspect we should have spaces but I don't feel strongly about it.

**Comment by Hwurzburg** [comment]
two commits...first updates the metadata separation, second makes all function names consistent (ie no spaces, and caps for each concatenated word in the function)....the latter was done with a careful replace all

**Review by peterbarker — APPROVED** [review]

**Comment by vierfuffzig** [comment]
this will require adjustments in param osd to respect per-vehicle parsing of servo functions
https://github.com/ArduPilot/ardupilot/blob/9a538ba15bd78d2040aab29f4ce3dfca83fa607d/libraries/AP_OSD/AP_OSD_ParamSetting.cpp#L121
ping @andyp1per

**Comment by andyp1per** [comment]
I don't think the values have changed here? Just the docs

**Pull request merged**